AnimatedTextInput/Classes/AnimatedTextView.swift

```diff
@@ -70,6 +70,11 @@ extension AnimatedTextView: TextInput {
         set { self.autocorrectionType = newValue }
     }
     
+    public var rightView: UIView? {
+        get { return nil }
+        set { }
```

Comment:
Remove setter

Reply:
UITextField public interface requires setter.
Yes, this code is unused in text view, but it's needed for AnimatedTextField